src/js/components/sharedComponents/Loading.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+// eslint-disable-next-line import/prefer-default-export
+export const LoadingWrapper = ({
+    msg = "Loading",
+    isLoading,
+    children
+}) => {
+    const [dots, updateDots] = useState("...");
+
+    useEffect(() => {
```

Comment:
why?

Reply:
B/c man we have dynamic loading dots. it's pretty sweet you should check it out.

Loading
Loading.
Loading..
Loading...
(repeat)

So we have to use setTime out and clean up.

Reply:
I did check it out.